### PR TITLE
V100.2.0

### PR DESCRIPTION
**Pull request by zinfin**
Hi @gunt0001 and @hansonchris, This PR updates the Android runtime SDK to 100.2.0.  Would you mind reviewing?

### CODE REVIEW

**Comment by gunt0001** [comment]
@fallsdownhill - please review.

File: maps-app/build.gradle [hunk 1]
```diff
@@ -28,15 +28,16 @@ android {
...
     // ArcGIS Android API
-    compile 'com.esri.arcgisruntime:arcgis-android:100.1.0'
+    implementation 'com.esri.arcgisruntime:arcgis-android:100.2.0'
     // square libs
-    compile 'com.jakewharton:butterknife:6.1.0'
+    implementation 'com.jakewharton:butterknife:8.8.1'
+    annotationProcessor 'com.jakewharton:butterknife-compiler:8.8.1'
 
     // Android Testing Support Library's runner and rules
     androidTestCompile ("com.android.support.test:runner:$rootProject.ext.runnerVersion"){
```
Comment (fallsdownhill) [inline maps-app/build.gradle]: I think all the androidTestCompiles should be replaced with androidTestImplementation

File: maps-app/README.md [hunk 2]
```diff
@@ -1,297 +0,0 @@
-Maps-App
```
Comment (fallsdownhill) [inline maps-app/README.md]: Why was this readme deleted?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # Maps-App Android
 
-This repo provides an example app for Android devices called Maps App that can be used as as starter for your organizations mapping app built with [ArcGIS Runtime SDK for Android](https://developers.arcgis.com/android/).  You can use the Maps App as is, or extend it using the ArcGIS Runtime SDK for Android.
+This repo provides an example app for Android devices called Maps App that can be used as as
```
Comment (fallsdownhill) [inline README.md]: "as" -> "a"

**Comment by zinfin** [comment]
@fallsdownhill Thanks, I've made the changes.

**Pull request merged**